packages/SwingSet/test/bundling/test-bundles.js

```diff
@@ -47,7 +47,7 @@ test('bundles', async t => {
   // We save this vat bundle (with 'disk()') to disk, to exercise
   // config.bundles.NAME.bundleSpec
   const diskBundle = await bundleSource(bfile('vat-disk.js'));
-  const diskBundleFilename = bfile('disk-bundle'); // in .gitignore
+  const diskBundleFilename = bfile('bundle-disk.js');
```

Comment:
Still in `.gitignore`, right? But top level now?

Reply:
Well, still gitignored but now by a glob. There's no longer an entry for the particular file, which is what I took the comment to mean. I expect our globs to be conventional enough that we don't have to indicate with comments what files will match.